amp/WEB-INF/src/org/digijava/kernel/ampapi/endpoints/publicportal/PublicPortalService.java

```diff
@@ -378,13 +385,51 @@ private static void applyFilterRules(Map<String, Object> filters, ReportSpecific
 
     public static PublicTotalsByMeasure getCountByMeasure(SettingsAndFiltersParameters config) {
         PublicTotalsByMeasure result = new PublicTotalsByMeasure();
+        config = addSomeMoreFiltersForValidatedProjects(config);
+
         result.getMeasure().put("original", "Total Activities");
         result.getMeasure().put("translated", TranslatorWorker.translateText("Total Activities"));
 
         result.setCount(PublicPortalService.getActivitiesCount(config != null ? config.getFilters() : null, false));
         return result;
     }
 
+    private static SettingsAndFiltersParameters addSomeMoreFiltersForValidatedProjects(SettingsAndFiltersParameters config) {
+        boolean fetchOnlyValidatedProjects = FeaturesUtil.getGlobalSettingValueBoolean(GlobalSettingsConstants.FETCH_ONLY_VALIDATED_PROJECTS);
```

Comment:
you are naming the setting validated projects, and you are filtering by ongoing projects. please rename the setting to OnGoinProjects